fullstop-plugins/fullstop-snapshot-source-plugin/src/main/java/org/zalando/stups/fullstop/plugin/snapshot/SnapshotSourcePlugin.java

```diff
@@ -78,18 +81,19 @@ public void processEvent(CloudTrailEvent event) {
 
             if (userData == null) {
                 violationSink.put(violationFor(event).withInstanceId(id).withPluginFullyQualifiedClassName(
-                                          SnapshotSourcePlugin.class).withType(MISSING_USER_DATA).build());
+                        SnapshotSourcePlugin.class).withType(MISSING_USER_DATA).build());
                 return;
             }
 
             String source = (String) userData.get(SOURCE);
             if (source == null) {
                 violationSink.put(violationFor(event).withInstanceId(id).withPluginFullyQualifiedClassName(
-                                          SnapshotSourcePlugin.class).withType(MISSING_SOURCE_IN_USER_DATA).build());
-            }
-            else if (source.matches(SNAPSHOT_REGEX)) {
+                        SnapshotSourcePlugin.class).withType(MISSING_SOURCE_IN_USER_DATA).build());
+            } else if (source.matches(SNAPSHOT_REGEX)) {
+                metaData.put("errorMessage", "EC2 should be deployed within docker and an immutable tag.");
```

Comment:
suggestion: "EC2 instances should be deployed with immutable Docker images only."